security/permissions.rst

```diff
@@ -0,0 +1,103 @@
+
+Permissions (AccessPolicy)
+--------------------------
+Firely Server supports filtering the requested client scopes by using built-in custom access policy resources. Both Smart-V1 and Smart-V2 scopes are supported. 
```

Comment:
Firely Server supports filtering the **granted** access scopes for a client by using built-in custom access policy resources.